docs/book/src/topics/failure-domains.md

```diff
@@ -130,6 +130,56 @@ spec:
 
 ```
 
+### Using Virtual Machine Scale Sets
+
+You can use an `AzureMachinePool` object to deploy a Virtual Machine Scale Set which automatically distributes VM instances across availability zones.
```

Comment:
let's make it more clear that you as a user need to set the available failure domains that you want to spread to in the MachinePool spec so it's not completely "autmatically". You can also use `az vm list-skus -l <location> --zone -o table` to list all the available zones per vm size in that location.

Reply:
I changed it, please take a look.